src/components/register-in-contest/register-additional-wrapper.jsx

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+import React from "react";
+import _ from "lodash";
+import { Input, ButtonInput } from "react-bootstrap";
+
+import RegisterAdditionalSearch from "./register-additional-search";
+import RegisterAdditionalConfirm from "./register-additional-confirm";
+
+
+export default React.createClass({
+  displayName: "RegisterAdditional",
+
+  getInitialState() {
+    return {
+      additionalEmail: "",
+      waitingForServerResponseAdditionalForm: false,
+      hasPerformedSearch: false,
+      additionalContestant: []
```

Comment:
Pentru ca e pe metoda de index si filtreaza. Stiu, m-am gandit sa returnam doar unul, dar era prea mult de modificat in API.